From a6ad3c55b34b76656ba496371278227a4db02992 Mon Sep 17 00:00:00 2001
From: Rui Loura <rui@joyent.com>
Date: Wed, 25 Jul 2018 18:21:45 +0000
Subject: [PATCH] TRITON-603 cnapi should be aware of alternate admin nictags
 (bump version) Reviewed by: Cody Peter Mello <cody.mello@joyent.com> Approved
 by: Cody Peter Mello <cody.mello@joyent.com>

---
 package.json | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/package.json b/package.json
index 6b55537..7b04baa 100644
--- a/package.json
+++ b/package.json
@@ -1,7 +1,7 @@
 {
   "name": "cnapi",
   "description": "SmartDataCenter Compute Node API",
-  "version": "1.10.0",
+  "version": "1.11.0",
   "author": "Joyent (joyent.com)",
   "private": true,
   "dependencies": {
-- 
2.21.0

